### PR TITLE
Issue 29 frontend make events clickable on calendar

### PR DESCRIPTION
**Pull request by RiddleMeDoo**
Waiting for [issue 23](https://github.com/DSC-UTMap/utmap-website/pull/39) pull request to resolve so that I can implement it into this branch.
- Made events on calendar clickable
- Clicking on the event will bring up a box with some data filled in
image

### CODE REVIEW

**Comment by RiddleMeDoo** [comment]
Current view:
image

**Review by Leila-U — APPROVED** [review]

File: utmap-client/src/components/pages/CalendarPage.js [hunk 1]
```diff
@@ -13,41 +13,79 @@ import {
...
+	location: 'Davis',
+	sublocation: 'Spiegal Hall',
+	organizer: 'Qianqian Feng'
+	},
+	{ startDate: '2021-01-27T12:00', endDate: '2021-01-29T13:30', title: 'Attend Andis Awesome Lecture' },
+	{ startDate: '2021-03-27T12:00', endDate: '2021-03-30T13:30', title: 'Get jacked' }
 ]
 
+function provideCustomAppointment(openEventInfo) {
+	//Curried component which overrides default event info popup
```
Comment (ShahmeerShahid) [inline utmap-client/src/components/pages/CalendarPage.js:40]: Wow curried component 👏

File: utmap-client/src/components/LocationData.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+//Location data will go here unless a better solution is found
```
Comment (ShahmeerShahid) [inline utmap-client/src/components/LocationData.js:1]: Good initiative

File: utmap-client/src/components/pages/CalendarPage.js [hunk 3]
```diff
@@ -13,41 +13,79 @@ import {
...
+	});
+}
+
 //Scheduler is the calendar, today, and taskbar components
-function CalendarPage() {
+function CalendarPage() { 
 	const [openEventForm, setOpenEventForm] = useState(false);
+	const [openEventInfo, setOpenEventInfo] = useState(false);
+	const [eventInfo, setEventInfo] = useState(
+		{ startDate: '2021-01-03T09:45', endDate: '2021-01-03T10:00', title: 'Blank' });
```
Comment (ShahmeerShahid) [inline utmap-client/src/components/pages/CalendarPage.js]: Are we sure we want this to be hardcoded?

Comment (RiddleMeDoo) [reply utmap-client/src/components/pages/CalendarPage.js]: That's true, I suppose we could have a blank object instead.

**Review by ShahmeerShahid — APPROVED** [review]

**Pull request merged**